features/reloading.feature

```diff
@@ -4,8 +4,7 @@ Feature:
   So that factory_bot_rails doesn't hold onto stale class references
 
   Scenario: Editing a model without editing the factory
-    When I successfully run `bundle exec rails new testapp -m ../../features/support/rails_template`
-    And I cd to "testapp"
+    When I create a new rails application
     And I add "factory_bot_rails" from this project as a dependency
     And I add "test-unit" as a dependency
```

Comment:
Since we remove `gem 'test-unit'` do we need it now?

Reply:
Yeah I think you are correct that we can remove this, since test-unit already ships with Ruby.